lib/elixir_bench/benchmarks/config.ex

```diff
@@ -15,6 +15,10 @@ defmodule ElixirBench.Benchmarks.Config do
       embeds_many :docker, Docker, primary_key: {:image, :string, []} do
         field :container_name, :string
         field :environment, {:map, :string}, default: %{}
+
+        embeds_one :wait, Wait, primary_key: false do
```

Comment:
I'd sort of expect `Wait` to be defined somewere but it seems to work same with `Docker` - slightly confusing though :thinking: